src/lakefs_spec/__init__.py

```diff
@@ -1,9 +1,14 @@
 from importlib.metadata import PackageNotFoundError, version
 
+from lakefs_client import __version__ as __lakefs_client_version__
+
 from .spec import LakeFSFile, LakeFSFileSystem
 
 try:
     __version__ = version("lakefs-spec")
 except PackageNotFoundError:
     # package is not installed
     pass
+
+lakefs_client_version = tuple(int(v) for v in __lakefs_client_version__.split("."))
```

Comment:
See my other comment:

```suggestion
lakefs_client_version = packaging.version.parse(__lakefs_client_version__)
```